src/journal/journald-server.c

```diff
@@ -2320,6 +2379,39 @@ int server_init(Server *s, const char *namespace) {
                         .interval = DEFAULT_KMSG_OWN_INTERVAL,
                         .burst = DEFAULT_KMSG_OWN_BURST,
                 },
+
+                .failed_entry_stats = {
+                        .without_rotate = {
+                                .error = 0,
+                                .num_entries = 0,
+                                .num_items = 0,
+                                .total_size = 0,
```

Comment:
no need to spell out zero initialization, that's the implied default.

Reply:
Fixed in commit https://github.com/systemd/systemd/commit/d77fdd883eeadce347ca517d21b8cf2b1a2a6f04